pages/poweredby.md

```diff
@@ -32,7 +32,7 @@ This website was made using:
 [commons]: http://commons.apache.org/
 [curator]: http://curator.apache.org/
 [hadoop]: http://hadoop.apache.org/
-[twill]: http://twill.incubator.apache.org/
+[twill]: http://twill.apache.org/
```

Comment:
These apache.org addresses should be https by default, unless there's a good reason not to. But, this can be done later.